CONTRIBUTING.md

```diff
@@ -78,12 +78,26 @@ When you're ready to contribute code to address an open issue, please follow the
 
     </details>
 
+    **Steps for setting up a conda env for tango:**
```

Comment:
this is the only real change in this doc

Reply:
Thanks for the whitespace fixes, but I actually don't think we should put this in here because:
1. It's platform specific. Will probably only work on OS X (maybe Linux if linxubrew is installed, but the `conda install pytorch..` line is definitely wrong).
2. It's brittle. It will easily break when the pytorch installation instructions go out of date (happens frequently), or if those test fixtures change.

Reply:
Though I do see how this is useful, so I suggest we put this right below line 70 above as an example, but remove all of the brittle parts. Maybe something like:

```
If you are new to Python, the easiest way to create a virtualenv suitable for working on Tango is with miniconda.
On Mac, you can install miniconda with via homebrew: `brew install miniconda`. Then you can create a new environment with `conda create -n tango python=3.9`. Don't forget to activate it with `conda activate tango`.
```

Reply:
i can add anything you want there...
maybe we can have a new md file fro this kind of thing...

i basically had to ask how to install.... since these steps were not clear to me

this is your app, you can add docs as you see fit... but this kind of info is needed if you ever want someone else at ai2 to contribute....

Reply:
Just a nit: Don't call it a "virtualenv" when it sets up a conda environment. "Virtualenv" is sort of a competitor to conda.

Reply:
@jonborchardt I'm making a PR now to improve the instructions. I'll ping you for a review.

Reply:
Please take a look at https://github.com/allenai/tango/pull/81